fontdrasil/src/orchestration.rs

```diff
@@ -160,9 +158,29 @@ impl<I: Identifier> Access<I> {
             Access::None => false,
             Access::Unknown => false,
             Access::All => true,
-            Access::One(allow) => id == allow,
-            Access::Set(ids) => ids.contains(id),
-            Access::Custom(f) => f(id),
+            Access::One(all) => all == id,
+            Access::Set(ids) => ids.iter().any(|allow| allow.check(id)),
+        }
+    }
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub enum AllOrOne<I: Identifier> {
+    All(I),
+    One(I),
```

Comment:
I noticed that in all the read/write_access method where you used this AllOrOne enum, you only ever used the AllOrOne:All variant, never the AllOrOne:One variant (except when matching all the enum variants).. I wonder what the latter is for. Is it just to logically counterpose the All variant, or is there a more concrete use for it that I am not seeing?

Reply:
Oh is it used implicitly when you do e.g. `AnyWorkId::Fe(FeWorkId::GlyphOrder).into()` because of the `impl From for AllOrOne` that returns an AllOrOne::One variant?

Reply:
Would it be possible to just have an 'AllGlyphs' or 'AllGvarFragments' work item, instead of trying to generalize over the whole idea of one/many?

Reply:
The only time you would want One is when you want something narrower than "all of this variant." Concretely, you want to block on a specific glyph not all glyphs. For example, create a One for every component to block on your components but not on every glyph in the world. I can add comments.

Reply:
I think AccessBuilder makes this clearer as now callsites are explict about matching a variant vs a specific instance.